apps/portal/components/pagesComponents/_videoEditorScreen/comments/CommentInput/CommentInput.tsx

```
@@ -42,7 +44,9 @@ const CommentInput: FC<ICommentInputProps> = ({
       />
 
       <div className={styles.commentEditButtons}>
-        <button onClick={() => handleCancel(comment.id)}>Cancel</button>
+        <button onClick={() => handleCancel(comment.id)}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)